### PR TITLE
Fix #1840 Upgrade to 1.8.4 fails with SQLite

### LINKED ISSUE #1840: Upgrade to 1.8.4 fails with SQLite
```SQLSTATE[HY000]: General error: 1 unrecognized token: "2fasecret"'```
SQLite/PgSQL seams not to support column names starting with a digit: http://stackoverflow.com/questions/3694276/what-are-valid-table-names-in-sqlite/3694305#3694305

http://community.mybb.com/thread-167214.html
@JN-Jones

### PR DESCRIPTION
**Pull request by Sama34**
Fix #1840 Upgrade to 1.8.4 fails with SQLite

### CODE REVIEW

File: install/resources/upgrade33.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * MyBB 1.8
+ * Copyright 2014 MyBB Group, All Rights Reserved
+ *
+ * Website: http://www.mybb.com
+ * License: http://www.mybb.com/about/license
+ *
+ */
+
+/**
+ * Upgrade Script: 1.8.1, 1.8.2, 1.8.3 or 1.8.4
```
Comment (Stefan-ST) [inline install/resources/upgrade33.php]: That should be just 1.8.4. upgrade32 is needed for 1.8.1-1.8.3.

Comment (Sama34) [inline install/resources/upgrade33.php]: Done.

**Comment by votanxerxes** [comment]
The file "install/resources/sqlite_db_tables.php" still had two errors.
(a missing and a wrong comma) Fixed version below.

27  	defaultviews TEXT NOT NULL, 
28  	loginattempts smallint unsigned NOT NULL default '0', 
29  	loginlockoutexpiry int unsigned NOT NULL default '0', 
30    authsecret varchar(16) NOT NULL default '', 
31    recovery_codes varchar(177) NOT NULL default '' 
32    );";

**Comment by Destroy666x** [comment]
Yep, thanks for noticing.

**Comment by Sama34** [comment]
Right.

**Pull request merged**